scripts/gen-iso.sh

```diff
@@ -42,8 +50,8 @@ sudo yum install -y genisoimage syslinux grub-tools createrepo_c
 
 cd /data
 
-sudo ./scripts/create-installimg.sh --srcurl http://mirrors.xcp-ng.org/8/${VERSION} -o ${NAMEIMG} ${VERSION}:${REPOSITORY}
+sudo ./scripts/create-installimg.sh --srcurl "https://updates.xcp-ng.org/8/${VERSION}" -o "${NAMEIMG}" "${VERSION}":"${REPOSITORY}"
 
-./scripts/create-install-iso.sh --netinstall --srcurl http://mirrors.xcp-ng.org/8/${VERSION} -V "${MNTVOL}" -o ${NAMEISONI} ${VERSION}:${REPOSITORY} ${NAMEIMG}
+./scripts/create-install-iso.sh --netinstall --srcurl "https://updates.xcp-ng.org/8/${VERSION}" -V "${MNTVOL}" -o "${NAMEISONI}" "${VERSION}":"${REPOSITORY}" "${NAMEIMG}"
 
-./scripts/create-install-iso.sh --srcurl http://mirrors.xcp-ng.org/8/${VERSION} -V "${MNTVOL}" -o ${NAMEISO} ${VERSION}:${REPOSITORY} ${NAMEIMG}
+./scripts/create-install-iso.sh --srcurl "https://updates.xcp-ng.org/8/${VERSION}" -V "${MNTVOL}" -o "${NAMEISO}" "${VERSION}":"${REPOSITORY}" "${NAMEIMG}"
```

Comment:
can be shorter with `"${VERSION}:${REPOSITORY}"`

Reply:
Yeah I went in the paranoid escape each variable direction without looking for optimising size.